test/micro/org/openjdk/bench/java/text/DefFormatterBench.java

```diff
@@ -52,15 +58,32 @@ public class DefFormatterBench {
 
     public double[] values;
 
+    private Date date;
+
+    private Object[] data;
+
+    private List<String> listData;
+
     @Setup
     public void setup() {
         values = new double[] {
             1.23, 1.49, 1.80, 1.7, 0.0, -1.49, -1.50, 9999.9123, 1.494, 1.495, 1.03, 25.996, -25.996
         };
+        date = new Date();
```

Comment:
Not sure how others feel, but I think it is probably best that the benchmark is a separate test file, as opposed to being added to an existing benchmark. Otherwise if we stick with this file, copyright needs to be updated.

Reply:
+1